model/registry/scm.yaml

```diff
@@ -0,0 +1,37 @@
+groups:
+  - id: registry.scm.repository
+    prefix: scm
+    type: attribute_group
+    brief: >
+     This group defines the attributes for source control management (SCM) systems.
+    attributes:
+      - id: repository.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the repository.
+        examples: ['opentelemetry-collector-contrib']
```

Comment:
This has been replaced by the URL. I still think having a name value would be useful. We currently use repository.name as an attribute in the `gitprovidwerreceiver` that allows us to look at metrics to a specific repository in a visualizer. Having the name is a nicer interface than a URL. Below is a screenshot of how we use this attributes for those metrics in a dashboard. It would be painful to the eyes if they all showed up as URLs. Maybe we can add this as an optional attribute on the next iteration.

https://github.com/open-telemetry/semantic-conventions/issues/1185

Reply:
This has been replaced with repository url full. However, `name` is still open for debate. Referenced in #1144